arviz/plots/loopitplot.py

```diff
@@ -0,0 +1,208 @@
+"""Plot LOO-PIT predictive checks of inference data."""
+import numpy as np
+import scipy.stats as stats
+import matplotlib.pyplot as plt
+from matplotlib.colors import to_rgb, rgb_to_hsv, hsv_to_rgb
+
+from ..stats import loo_pit as _loo_pit
+from .plot_utils import _scale_fig_size
+from .kdeplot import _fast_kde
+from .hpdplot import plot_hpd
+
+
+def plot_loo_pit(
+    idata=None,
+    y=None,
+    y_hat=None,
+    log_weights=None,
+    ecdf=False,
+    ecdf_fill=True,
+    n_unif=100,
+    use_hpd=False,
+    credible_interval=0.94,
+    figsize=None,
+    textsize=None,
+    color="C0",
+    legend=True,
+    ax=None,
+    plot_kwargs=None,
+    plot_unif_kwargs=None,
+    hpd_kwargs=None,
+    fill_kwargs=None,
+):
+    """Plot Leave-One-Out (LOO) probability integral transformation (PIT) predictive checks.
+
+    Parameters
+    ----------
+    idata : InferenceData
+        InferenceData object.
+    y : array, DataArray or str
+        Observed data. If str, idata must be present and contain the observed data group
+    y_hat : array, DataArray or str
+        Posterior predictive samples for ``y``. It must have the same shape as y plus an
+        extra dimension at the end of size n_samples (chains and draws stacked). If str or
+        None, idata must contain the posterior predictive group. If None, y_hat is taken
+        equal to y, thus, y must be str too.
+    log_weights : array or DataArray
+        Smoothed log_weights. It must have the same shape as ``y_hat``
+    ecdf : bool, optional
+        Plot the difference between the LOO-PIT Empirical Cumulative Distribution Function
+        (ECDF) and the uniform CDF instead of LOO-PIT kde.
+        In this case, instead of overlaying uniform distributions, the beta ``credible_interval``
+        interval around the theoretical uniform CDF is shown. This approximation only holds
+        for large S and ECDF values not vary close to 0 nor 1. For more information, see
+        `Vehtari et al. (2019)`, `Appendix G <https://avehtari.github.io/rhat_ess/rhat_ess.html>`_.
```

Comment:
Do you think this explanation will be enough or should I add a warning when `ecdf=True` and the number of observations is below some number?